### DISCUSSION TITLE
Importing Data from Stable to Lazer

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#13548] [Ideas] [answered]
Greetings, I was wondering if we could import data from stable to lazer in a more efficient method. Obviously, I play on the stable client and here and there I like to test out the newly added features in lazer. When doing so, I tend to import all of my data into lazer and the time it takes for data to transfer especially in beatmaps and skins takes way too much time. I believe this is because it checks each individual folder and imports them, regardless if they are already there. For the sake of efficiency, could there be a system that checks the already existing data in the lazer client and skips past it? I like to check lazer like once every few weeks and I like to try out the new maps and skins that I have used in stable but the process takes well over 30 minutes each time, and it is mostly due to it adding EVERY single piece of data when you can simply check if it is already there. Anyways, not a big issue, just an idea to make lazer a better experience when switching from stable. Thank you in advance!

**Comment by peppy** [comment, marked as answer]
There is no more efficient method. This is being tracked here https://github.com/ppy/osu/issues/12746.
